src/main/java/com/box/sdk/BoxJSONResponse.java

```diff
@@ -31,33 +34,60 @@ public BoxJSONResponse(HttpURLConnection connection) {
     }
 
     /**
+     * Constructs a BoxAPIResponse with an http response code and response body.
+     * @param responseCode http response code
+     * @param httpHeaders map of http headers
+     * @param body response body as Json Object
+     */
+    public BoxJSONResponse(int responseCode, Map<String, String> httpHeaders, JsonObject body) {
+        super(responseCode, httpHeaders);
+        this.jsonObject = body;
+        this.json = body.toString();
+    }
+
+    /**
+     * Get response as Json Object.
+     * @return response as JsonObject
+     */
+    public JsonObject getJsonObject() {
```

Comment:
hm?

Reply:
Java SDK deals with strings instead of JsonObject making it hard for devs who want to read contents in a response that's JSON, unless it's cast to a resource element. This method gives that feature